README.md

```diff
@@ -12,7 +12,7 @@
 
 <p align="center">
   <a href="https://circleci.com/gh/CityOfZion/dora">
-    <img src="https://circleci.com/gh/CityOfZion/dora.svg?style=svg&circle-token=a7d4029776ee0262fce4c3aa466f329ae6616e5d">
```

Comment:
I don't think the token is required as this is not a private repository

https://circleci.com/docs/status-badges/#creating-badges-for-private-repositories